target/linux/ramips/dts/mt7620a_wavlink_wl-wn535k1.dts

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "wavlink,wl-wn535k1", "ralink,mt7620a-soc";
+	model = "Wavlink WL-WN535K1";
+
+	aliases {
+		label-mac-device = &wifi0;
+		led-boot = &led_status_blue;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_red;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		touchlink {
+			label = "touchlink";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
```

Comment:
So this is kind of touch sensor? Interesting...
Is it used for WPS in the stock firmware as well?

Reply:
Actually it's very strange.
![image](https://user-images.githubusercontent.com/2470733/148233913-0900d594-0fc1-4af5-a7cc-a5193751a9fd.png)
If I touch the top side of the router (the blue one) the gpio gets triggered. However if I try to press it, it doesn't make any "clicking" sound and the plastic doesn't move down, so it's not a classic "push" button.
In conclusion I think that this is just a capacitive button.

Honestly I didn't test it, but according to the OEM website it should behave like a WPS button:
![image](https://user-images.githubusercontent.com/2470733/148236351-2fbc3b99-1178-4ad4-8466-6394c80bc6c0.png)